test/integration/mesos/mesos_helpers.bash

```diff
@@ -2,11 +2,36 @@
 
 load ../helpers
 
-MESOS_CLUSTER_ENTRYPOINT=${MESOS_CLUSTER_ENTRYPOINT:-0.0.0.0:5050}
+MESOS_IMAGE=jimenez/mesos-dev:clang
+MESOS_MASTER_PORT=$(( ( RANDOM % 1000 )  + 7000 ))
+BASE_PORT=2375
+
+# Start mesos master and slave.
+function start_mesos() {
+	MESOS_MASTER=$(
+	docker_host run -d --name mesos-master --net=host \
+	$MESOS_IMAGE /mesos/build/bin/mesos-master.sh --ip=127.0.0.1 --work_dir=/ --registry=in_memory --port=$MESOS_MASTER_PORT
+		)
+
+	MESOS_SLAVE=$(
+	docker_host run -d --name mesos-slave --volumes-from node-0 -e DOCKER_HOST="${HOSTS[0]}" -v /sys/fs/cgroup:/sys/fs/cgroup --net=host \
```

Comment:
Also one thing I usually do is to mount in the sandbox volume from the host, so like -v /tmp/mesos:/tmp/mesos
Doing this we can also try out restarting the slave and it should recover all the docker tasks with swarm, etc.